core/src/test/scripts/system_tests_validate_data.py

```diff
@@ -10,6 +10,7 @@
 import logging
 import os
 from importer import validateData
+from importer.validateData import get_pom_path
```

Comment:
Is there a reason against accessing this function via the import on the line above, as `validateData.get_pom_path`? I think that would more explicitly show you're overriding (mocking) a global in another module.